pkg/objstore/minio/store.go

```diff
@@ -121,6 +133,17 @@ func (s store) Open(name string) (io.ReadSeeker, error) {
 	})
 }
 
-func (s *store) Healthcheck(ctx context.Context) error {
+func (s *store) Healthcheck(_ context.Context) error {
 	return nil
 }
+
+// GenerateBucketName return bucket name based on path prefix, separator, bucketName
+func GenerateBucketName(pathPrefix, separator, name, componentName string) (out string) {
```

Comment:
please explain why are you mixing bucket and path concepts here? 

We've discussed approach where 2 env variables (options) would be introduced 
`MINIO_BUCKET_NAME`
`MINIO_PATH_PREFIX`

Both would go through simple string replacement, replacing `{component}` (if it would exist of corse) with system|compose|....

Then the both options would be passed to the client where bucket would be used same as today and path-prefix would be applied within save/remove/open.

What am I missing here?

Reply:
No, you are right, my bad I will fix it.